src/goose/console.clj

```diff
@@ -1,60 +1,63 @@
 (ns goose.console
+    "Functions to serve the console web interface"
   (:require [clojure.string :as str]
             [goose.broker :as b]
+    [goose.defaults :as d]
             [hiccup.page :refer [html5 include-css include-js]]
             [ring.middleware.keyword-params :as ring-keyword-params]
             [ring.middleware.params :as ring-params]
             [ring.util.response :as response]))
 
 ;; Page handlers
-(defn load-css
-  "Load static css file"
+(defn ^:no-doc load-css
+      "Load goose's static css file"
```

Comment:
question: Shouldn't this be "Goose console's static ..."?

Similar comment for the `load-js`'s docstring as well